src/ItemOverlay/ItemOverlay.js

```diff
@@ -84,8 +84,10 @@ var ItemOverlay = module.exports = kind(
 	/**
 	* @private
 	*/
-	autoHideChanged: function() {
-		this.addRemoveClass('auto-hide', this.get('autoHide'));
+	autoHideChanged: function(was, is) {
+		if (this.autoHide) {
+			this.set('showing', (was || this.spotFocused));
```

Comment:
I'm not quite sure I follow where `this.spotFocused` is coming from and why it is being checked here.